foyer/forcefield.py

```diff
@@ -385,6 +403,24 @@ def included_forcefields(self):
             self._included_forcefields[basename] = ff_filepath
         return self._included_forcefields
 
+    def _parse_version_number(self, forcefield_file):
+        with open(forcefield_file, 'r') as f:
+            tree = ET.parse(f)
+            root = tree.getroot()
+            try:
+                return root.attrib['version']
+            except KeyError:
```

Comment:
Going off of Mike's comment about raising a warning, this seems like the place where that would go.
```suggestion
            except KeyError:
                warn.Warn("version warning message here")
                return None
```